float64 correlation
landmark_extractor/FloorPlanCoordMsg[] chain